components/checkout/ShippingAddressSection.tsx

```diff
@@ -83,6 +85,11 @@ export const ShippingAddressSection = ({
         <>
           {editing ? (
             <>
+              <SavedAddressSelectionList
```

Comment:
Render component only if a user is authenticated (ATM code fire API call even if checkout is anonymous)